doc/source/timeseries.rst

```diff
@@ -868,6 +868,26 @@ normalized after the function is applied.
    hour.apply(pd.Timestamp('2014-01-01 23:00'))
 
 
+.. _timeseries.dayvscalendarday:
+
+Day vs. CalendarDay
+~~~~~~~~~~~~~~~~~~~
+
+:class:`Day` (``'D'``) is a timedelta-like offset that respects absolute time
+arithmetic and essentially is an alias for 24 :class:`Hour`. This offset is the default
```

Comment:
btw do we test that this is true? I think you are right, so maybe change to the language to say it *is* equivalent? (or note the differences)